roles/ceph-rgw/handlers/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: update period
-  command: radosgw-admin period update --commit
+  command: radosgw-admin --cluster {{ cluster }} period update --commit
```

Comment:
this command will be run on first mon node as many rgw nodes there is on the cluster, is that expected ?